luigi/hdfs.py

```diff
@@ -308,7 +308,7 @@ def get_bite(self):
         If Luigi has forked, we have a different PID, and need to reconnect.
         """
         if self.pid != os.getpid() or not self._bite:
-            client_kwargs = dict(filter(lambda (k, v): v is not None and v != '', {
+            client_kwargs = dict(filter(lambda k_v: k_v[1] is not None and k_v[1] != '', {
```

Comment:
I really don't like treating tuples like arrays and using indices. Much nicer to unpack into variables